README.md

```diff
@@ -49,9 +48,9 @@ To create new projects, use the `brigade-project` Helm chart. While inside the G
 repository cloned above, run these commands:
 
 ```console
-$ helm inspect values ./charts/brigade-project > myvalues.yaml
+$ helm inspect values brigade/brigade-project > myvalues.yaml
 $ # edit myvalues.yaml
```

Comment:
at some point, let's flesh this out by breaking this out into a section saying _what they actually do_. yes, there is information in the file; no, it's just reference information and does not necessarily have anything to do with getting hello brigade working. So here, I'd hope we'd say, "if you're just going to test it out using the hello brigade experience, you do not change anything in the file at all." Instead, we tell them to edit things, but if you look at the file, for the hello world experience, you should NOT do anything. None of it is used.

Reply:
Okay. Does that level of detail go in the README or in the install.md document?